### DISCUSSION TITLE
Update SynEdit from Lazarus git

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1524] [Suggestions]
lazarus git has 2 big fixes

- highlighter for HTML don't hang on 700k length one-liner HTML
- JSON (ie JS) highlighter is fixed to handle escaped dbl-quote inside string  value: `"key": "val\"val2",`

**Comment by alexx2000** [comment]
1. Lazarus 3.2 has same HTML highlighter. Lazarus 3.2 already in use to build DC under several distro.
2. DC does not use Lazarus JSON highlighter. Latest DC JSON highlighter supports escaped double-quote inside string.
